pyproject.toml

```
@@ -121,7 +121,7 @@ add-ignore = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--cov=caf.base --cov-report=xml --doctest-modules -n auto"
+#addopts = "--cov=caf.base --cov-report=xml --doctest-modules -n auto"
```

This shouldn't be commented out
```suggestion
addopts = "--cov=caf.base --cov-report=xml --doctest-modules -n auto"
```